static/sass/_pattern_resources.scss

```diff
@@ -0,0 +1,34 @@
+.p-card--resources {
+  background: #000 url("https://assets.ubuntu.com/v1/134d6bc1-video-background-suru.jpg");
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  height: 338px;
+  max-width: 567px;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  position: relative;
+
+  h2 {
+    color: white;
```

Comment:
Can you add `color` to the `.p-card--resources` element. If you set the background we should set color and we are not guaranteed to have h2 is all use cases.